src/elements/OCAlert.vue

```diff
@@ -43,6 +43,16 @@ export default {
       return classes
     },
   },
+  methods: {
+    onClose(val) {
```

Comment:
hmm not sure I need the val at all here right?

Reply:
should be the event argument

Reply:
yes it is